versions/development/SPEC.md

```diff
@@ -2138,44 +2138,280 @@ Now everything inside of `tasks.wdl` must be accessed through the default namesp
 Each namespace may contain namespaces, tasks, and at most one workflow.  The names of the contained namespaces, tasks, and workflow need to be unique within that namespace. For example, one cannot import two workflows while they have the same namespace identifier. Additionally, a workflow and a namespace both named `foo` cannot exist inside a common namespace. Similarly there cannot be a task `foo` in a workflow also named `foo`.
 However, you can import two workflows with different namespace identifiers that have identically named tasks. For example, you can import namespaces `foo` and `bar`, both of which contain a task `baz`, and you can call `foo.baz` and `bar.baz` from the same primary workflow.
 
-# Scope
+# Value Scopes and Reachability
 
-Scopes are defined as:
+## Top Level Scopes
 
-* `workflow {...}` blocks
-* `call` blocks
-* `if(expr) {...}` blocks
-* `scatter(x in y) {...}` blocks
+The top level scopes which can contain value declarations are:
 
-Inside of any scope, variables may be [declared](#declarations).  The variables declared in that scope are visible to any sub-scope, recursively.  For example:
+* `workflow {...}` definitions
+* `task {...}` definitions
+
+Within these top-level scopes, various sub-sections exist which can also contain value declarations:
+
+* `scatter` blocks
+* `if` blocks
+* `input` sections
```

Comment:
This is probably not important for this PR, but it's unclear what the difference between blocks and sections are. The text also uses `section` to refer to sections of the document itself, so using `block` everywhere might be better. Unless there's a semantic difference between blocks and sections.

Reply:
Fair enough. In my head I use the two interchangeably but I'm happy to prefer `block` over section for the sake of clarity